app/src/main/res/xml/preferences_master.xml

```diff
@@ -583,6 +583,19 @@
 
         </PreferenceScreen>
 
+        <!-- asciidoc Screen -->
+        <PreferenceScreen
+            android:icon="@drawable/asciidoc_icon_black_24dp"
+            android:title="@string/asciidoc">
+
+            <PreferenceCategory android:title="@string/actionbutton">
+                <Preference
```

Comment:
![grafik](https://user-images.githubusercontent.com/6735650/194112509-713ecbaa-caa9-4451-a12e-7fc83ff08871.png)

Please don't add png's where there is no must. Use vector icons, like see the other icons used at buttons, for example `ic_baseline_keyboard_arrow_up_24.xml` is. For that similary to the `AsciiDoc format icon`, please use Material icons. All material icons are for granted available in that xml/vector format in that size required. 

<https://fonts.google.com/icons?icon.query=overline&icon.platform=android>

![grafik](https://user-images.githubusercontent.com/6735650/194113267-f03da45d-9be9-4d78-a1d7-6957d1cdc453.png)

![grafik](https://user-images.githubusercontent.com/6735650/194113302-c12d7cec-a267-4d2d-85fa-f3695768ab68.png)

Reply:
I copied them from one zip file. But will them remove and keep only the SVG xml.
I tried to find Material icons, but I did not found for all requirements. So if there are better ideas for icons: I used the best I found, which are free to use.

Reply:
You can leave it for later stage of the merge request if you prefer. But merging something with unknown license, _copied from somewhere of the web_ isn't something that can be accepted in open source projects (or more even, in any publicized software 😄 ).

For the majority I guess you can find already an icon in-place, like bold for example. So just browse settings, try all formats and see the icons used. You can then look it up in the code what R.drawable id is behind it.  
I myself don't do it any different when I'm on adding new feature and need a icon.

Reply:
You can leave it for later stage of the merge request if you prefer. But merging something with unknown license, _copied from somewhere of the web_ isn't something that can be accepted in open source projects (or more even, in any publicized software 😄 ).

For the majority I guess you can find already an icon in-place, like bold for example. So just browse settings, try all formats and see the icons used. You can then look it up in the code what R.drawable id is behind it.  
I myself don't do it any different when I'm on adding new feature and need a icon.

And for this specific overline example, as you can see there is a easy way to get the svgish-xml in a really easy way, and you can be sure it's license wise totally fine to add.